pandora-client-web/src/components/directMessage/directMessage.tsx

```diff
@@ -100,13 +101,15 @@ function DirectChannelInput(): ReactElement | null {
 		ref.current?.focus();
 	}, [channel.account]);
 
+	const actualRef = useTextFormatting(ref);
+
 	if (!channel.account) {
 		return null;
 	}
 
 	return (
 		<textarea
-			ref={ ref }
+			ref={ actualRef }
```

Comment:
Little nitpick: You use it once directly and once indirectly in the files where I placed this comment.